apps/challenges/admin.py

```diff
@@ -1,3 +1,52 @@
 from django.contrib import admin
 
-# Register your models here.
+
+from .models import Challenge, Phase
+from base.admin import TimeStampedAdmin
+
+
+@admin.register(Challenge)
+class ChallengeModelAdmin(TimeStampedAdmin):
+    list_display = ("title",
+                    "description",
+                    "terms_and_conditions",
+                    "submission_guidelines",
+                    "evaluation_details",
+                    "image",
+                    "start_date",
+                    "end_date",
+                    "creator",
+                    "published",
+                    "enable_forum",
+                    "anonymous_leaderboard")
+    list_filter = ("title",
+                   "description",
+                   "terms_and_conditions",
+                   "submission_guidelines",
+                   "evaluation_details",
+                   "image",
+                   "start_date",
+                   "end_date",
+                   "creator",
+                   "published",
+                   "enable_forum",
+                   "anonymous_leaderboard")
+    search_fields = ("title",
+                     "description",
+                     "terms_and_conditions",
+                     "submission_guidelines",
+                     "evaluation_details",
+                     "image",
+                     "start_date",
+                     "end_date",
+                     "creator",
+                     "published",
+                     "enable_forum",
+                     "anonymous_leaderboard")
+
+
+@admin.register(Phase)
+class PhaseModelAdmin(TimeStampedAdmin):
+    list_display = ("name", "description", "leaderboard_public", "challenge")
```

Comment:
remove description. reason : it is a text field hence width of row will be larger and a slight inconvenience when looking at the dataset.